app/uk/gov/hmrc/hec/services/TaxCheckService.scala

```diff
@@ -70,6 +80,16 @@ class TaxCheckServiceImpl @Inject() (
     taxCheckStore.store(taxCheck).map(_ => taxCheck)
   }
 
+  def updateTaxCheck(
+    taxCheck: HECTaxCheck
+  )(implicit hc: HeaderCarrier): EitherT[Future, Error, HECTaxCheck] =
+    taxCheckStore.store(taxCheck.copy(isExtracted = true)).map(_ => taxCheck)
+
+  def updateAllHecTaxCheckStatus(list: List[HECTaxCheck])(implicit
```

Comment:
this doesn't look like it's specific to the extraction status so we could name this to something like `updateAllHecTaxChecks` I think

Reply:
renamed